.github/workflows/deployment-ci.yml

```diff
@@ -74,14 +74,7 @@ jobs:
       - name: Build with Gradle
         uses: gradle/gradle-build-action@v2
         with:
-          arguments: --stacktrace --info build -x apiCheck
```

Comment:
we want apiCheck to happen on releases

Reply:
it will always be executed, it's part of the `check` pipeline, see https://github.com/Kotlin/binary-compatibility-validator#tasks